### ISSUE TITLE
Showing changes in the original file

### ISSUE THREAD

**Issue by dluschan** [gitlocalize/feedback#127]
### Summary

Any change to the text in the original file requires the translation of an entire paragraph again.

### Steps to reproduce

Somehow change the original file, for example, put a comma in it.

### Repository URL

[Comma added.](https://github.com/rust-lang-ru/book/commit/19e6285a28ccdd02af58f3c44dfb7186e9421fc8)

### What is the current *bug* behavior?

The paragraph needs to be translated again on GitLocalize.

### What is the expected *correct* behavior?

It should be visible what has changed in the original and the last translation of this file location.

**Comment by ilyaspiridonov** [comment]
@dluschan This sounds like the correct behaviour, but you should also be able to see (and apply) the previous translation in the Translation Memory suggestions at the bottom of the screen. Can you see that?

**Comment by dluschan** [comment]
Yes, I can see the previous translation in the Translation Memory suggestions at the bottom. The problem isn’t reusing it. The issue is that the UI doesn’t show what exactly changed in the source. Even a one-character edit (e.g., a comma) marks the whole paragraph as changed, but there’s no diff highlighting. Please show a visible diff between the previous and current source so I can adjust only the affected part.

<img width="1306" height="803" alt="Image" src="https://github.com/user-attachments/assets/6892f02c-aa63-471d-85f1-261ae5119147" />
